notebook/_version.py

```diff
@@ -1,2 +1,28 @@
-version_info = (4, 1, 0, 'b1')
+"""
+store the current version info of the notebook.
+
+"""
+
+# Downstream maintainer, when running `python.setup.py jsversion`,
+# the version string is propagated to the JavaScript files,  do not forget to
+# patch the JavaScript files in `.postN` release done by distributions. 
+
+# Next beta/alpha/rc release: The version number for beta is X.Y.ZbN **without dots**. 
+
+version_info = (4, 1, '0b1')
```

Comment:
> Essentially, the only fix to care about is that setuptools-free sdist upload uses the same string for prereleases as the coerced/normalized version bdist_wheel gets from setuptools.

I would still like the version from PyPI to match the notebook --version. I can make `__version__` match and keep version_info a 4 tuple.